### DISCUSSION TITLE
Select tree item in variables view programatically

### DISCUSSION THREAD

**Discussion by SWETAS04** [microsoft/vscode-discussions#295] [Extension Development QnA] [answered]
In our extension , we want to add a context menu for "Go to Variable" in  Variables Tree for the variable being selected in the editor .
It should expand the Tree and reveal the variable.
I did not  find any suitable vscode API to access the Variables View Tree to get this done.
Is there any way to achieve this?

**Comment by isidorn** [comment, marked as answer]
There is no way to access the Variables Tree using the extensions. You might be able to contribute actions to it. And you might be able to somehow expand that tree using some command. But I think it is not possible to reveal a variable.

fyi @roblourens
